lib/smart_proxy_remote_execution_ssh/runners/script_runner.rb

```diff
@@ -331,20 +331,22 @@ def run_sync(command, stdin = nil)
       stderr = ''
       exit_status = nil
 
+      options = [
+        "-o User=#{@ssh_user}",
+        "-o Port=#{ssh_options[:port]}",
+        "-o IdentityFile=#{ssh_options[:keys][0]}",
+        "-o IdentitiesOnly=#{ssh_options[:keys_only]}",
+        "-o StrictHostKeyChecking=accept-new",
+        "-o PreferredAuthentications=#{ssh_options[:auth_methods].join(',')}",
+        "-o UserKnownHostsFile=#{ssh_options[:user_known_hosts_file] ? ssh_options[:user_known_hosts_file] : ''}",
```

Comment:
It would be better to not pass the option at all if we don't have a value for it